usaspending_api/download/v2/views.py

```diff
@@ -41,7 +41,7 @@
 
 @api_transformations(api_version=settings.API_VERSION, function_list=API_TRANSFORM_FUNCTIONS)
 class BaseDownloadViewSet(APIDocumentationView):
-    s3_handler = S3Handler(name=settings.BULK_DOWNLOAD_S3_BUCKET_NAME, region=settings.BULK_DOWNLOAD_AWS_REGION)
+    s3_handler = S3Handler(name=settings.BULK_DOWNLOAD_S3_BUCKET_NAME)
```

Comment:
Will this fall back to the env var?

Reply:
Yes, if you look at S3Handler's `init` (not sure why we are using such a simple wrapper for boto) it is set there.